content/news/2023-07-09-bevy-0.11/index.md

```diff
@@ -291,7 +291,7 @@ differently, we get those awkward specular highlights. With parallax mapping,
 they are gone.
 
 <b style="display:block; margin-bottom: -18px">Drag this image to compare</b>
-<div class="image-compare" style="aspect-ratio: 16 / 9" data-title-a="Normals Only" data-title-b="Parallax+Normals">
+<div class="image-compare" style="aspect-ratio: 16 / 9; --slider-value: 39%" data-title-a="Normals Only" data-title-b="Parallax+Normals">
```

Comment:
This just improves the utility of this image when js is disabled I think
before (js disabled):
![image](https://github.com/bevyengine/bevy-website/assets/17050131/7ecd3fbf-bf66-48bd-bf6f-462955862da6)
after (js disabled):
![image](https://github.com/bevyengine/bevy-website/assets/17050131/2199c10b-0219-4192-a8d7-8223a0259777)